tests/performance/plugins/server-timing/mu-plugin.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+add_action(
+	'template_redirect',
+	function() {
+		ob_start();
+
+		global $server_timing_values, $timestart;
+
+		if ( ! is_array( $server_timing_values ) ) {
+			$server_timing_values = array();
+		}
+
+		$server_timing_values['before-template'] = microtime( true ) - $timestart;
+
+		add_action(
+			'shutdown',
+			function() {
+
+				global $server_timing_values, $timestart;
+
+				if ( ! is_array( $server_timing_values ) ) {
+					$server_timing_values = array();
+				}
```

Comment:
This shouldn't be needed strictly speaking, since the variable is also ensured to be an array above. If it wasn't an array here, it also would be missing the `before-template` key which is unconditionally checked further below.

Reply:
Yes. Make sense.